universal-application-tool-0.0.1/app/modules/SecurityModule.java

```diff
@@ -186,7 +186,7 @@ protected Config provideConfig(
     if (adClient != null) {
       clientList.add(adClient);
     }
-    if (URI.create(this.baseUrl).getHost().equals("localhost")) {
+    if (FakeAdminClient.canEnable(URI.create(baseUrl).getHost())) {
```

Comment:
Thoughts (non-blocking): slightly surprised this is not enabled in our browser tests docker environment so far. What does it do? Why our browser tests don't seem to have problems with it?

Reply:
I think it is enabled there - I think that's also `localhost`.

Reply:
I see. `baseUrl` is from config not the request so it is still `localhost`. Make sense.